src/hooks/useSwapAdjustedAmounts.ts

```diff
@@ -39,7 +39,7 @@ export default function useSwapAdjustedAmounts(tradeDetails: Quote) {
   const address = inputAsExact
     ? outputCurrency.mainnet_address || outputCurrency.address
     : inputCurrency.mainnet_address || inputCurrency.address;
-  const priceValue = genericAssets[address]?.price?.value ?? 0;
+  const priceValue = genericAssets[address]?.price?.value ?? 0.0000000001;
```

Comment:
wondering what's this number coming from?

Reply:
me too

Reply:
lmao look away I meant to change this

Reply:
was a placeholder for token price, replaced with the correct way to get: `ethereumUtils.getAssetPrice`